test/libraries/cufft.jl

```diff
@@ -342,3 +342,10 @@ end
 
     @test Array(dy) ≈ y
 end
+
+@testset "CUDA.jl#2409" begin
+    x = CUDA.zeros(ComplexF32, 4)
+    p = plan_ifft(x)
+    @test p isa AbstractFFTs.ScaledPlan
+    @test convert(CUFFT.cufftHandle, p) === convert(CUFFT.cufftHandle, p.p)
```

Comment:
Maybe instead of testing this specific handle conversion (which is likely to break when we, e.g., rename the internal `cufftHandle`) just pass the scaled plan to an API call that performs the conversion? Assuming that's something that currently doesn't work, and that this PR fixes.

Reply:
I added a test that calls `CUFFT.cufftGetSize` for a `ScaledPlan`.

Reply:
Ok, thanks. This wasn't the test I expected, but I guess you're actually using these ScaledPlan-wrapped CUFFT plans using low-level API calls? Ie., existing high-level functionality (like `mul!`) already works fine with scaled CUFFT plans?

Reply:
Yes, I have several large FFTs that run sequentially (with other processing in between) so I share the "work area" of the largest plan across all of the FFT plans to reduce the memory overhead.  `cufftGetSize` is the most innocuous of the methods I call and it exercises the new conversion method when passed a `ScaledPlan` so I went with that method for the test.  Thanks for merging this!